app/audit/models/SwitchReportingMethodAuditModel.scala

```diff
@@ -38,10 +39,10 @@ case class SwitchReportingMethodAuditModel(journeyType: String,
   override val detail: JsValue = {
     Utilities.userAuditDetails(user) ++
```

Comment:
please remove usage of  messagesApi: MessagesApi from this model as this is not required and creates more problem with support in the future, we really don't need this type of dynamic logic here